gulpfile.js

```diff
@@ -95,9 +95,18 @@ gulp.task("yarn:build", () => {
 });
 
 /**
+ * Run the lint task
+ */
+gulp.task("yarn:lint", () => {
+  return gulp.src(TYPESCRIPT_SOURCE_DIR)
+    .pipe(exec(`yarn run lint`))
+    .pipe(exec.reporter());
+});
+
+/**
  * Run the test task
  */
-gulp.task("jest:test", () => {
+gulp.task("jest:test", ["yarn:lint"], () => {
```

Comment:
I would just name it `test`, or at least have an alias just named `test`.

Also worth updating [this paragraph](https://github.com/teamdigitale/digital-citizenship-functions#unit-tests) with instructions on how to run the tests.

Reply:
I've refactored the script a little to make room for other kinds of test (other than unit) in future.